README.md

```diff
@@ -1,96 +1,154 @@
 # Swift Numerics
   
 ## Introduction
-Swift Numerics provides a set of modules that support numerical computing in Swift.
-These modules fall broadly into two categories:
 
-- API that is too specialized to go into the standard library, but which is sufficiently general to be centralized in a single common package.
-- API that is under active development toward possible future inclusion in the standard library.
+Swift Numerics provides a set of modules that support numerical computing in
+Swift. These modules fall broadly into two categories:
 
-There is some overlap between these two categories, and API that begins in the first category may migrate to the second as it matures and new uses are discovered.
+- API that is too specialized to go into the standard library, but which is
+  sufficiently general to be centralized in a single common package.
+- API that is under active development toward possible future inclusion in the
+  standard library.
+
+There is some overlap between these two categories, and API that begins in the
+first category may migrate into the second as it matures and more users start
+using Swift Numerics.
+
+Swift Numerics modules are fine-grained. For example, if you need support for
+Complex numbers, you can import ComplexModule¹ as a standalone module:
 
-Swift Numerics modules are fine-grained; if you need support for Complex numbers, you can import ComplexModule¹ without pulling in everything else in the library as well:
 ```swift
 import ComplexModule
 
 let z = Complex<Double>.i
 ```
-However, there is also a top-level `Numerics` module that simply re-exports the complete public interface of swift-numerics:
+
+There is also a top-level `Numerics` module that re-exports the complete public
+interface of swift-numerics:
+
 ```swift
 import Numerics
 
-// All swift-numerics API is now available
+// The entire `swift-numerics` API is now available
```

Comment:
`swift-numerics` doesn't seem to be appropriate formatting here; "swift-numerics" is not code. In prose we usually style the project "Swift Numerics".

Reply:
Agreed, thanks @stephentyrone